### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by 111phil** [OpenSlides/OpenSlides#6364] [Ideas]
We were contacted by participants with disabilities that it is quite hard for them to use OpenSlides, since it is not compatible with assistive technologies. This should be enhanced in the future.

**Comment by ostcar** [comment]
I think this is a very important point. I created this issue to track the progress. I don't develop the client. But I think it could be helpful, if you could ask your participants, if they can give us some hints.

https://github.com/OpenSlides/openslides-client/issues/856
